### ISSUE TITLE
Can't login via web (M_UNKNOWN)

### ISSUE THREAD

**Issue by akater** [vector-im/riot-web#6416]
### Description

Can't login via web, for unclear reasons.

### Steps to reproduce

- Go to https://riot.im/app/#/login
- Accept cookies, turn on JS, allow requests to matrix.org, allow same for matrix org for good measure
- Provide login and password for a “matrix.org Matrix ID” login option

The web client says:
“Error: Problem communicating with the given homeserver. (M_UNKNOWN)”

### Version information

- **Browser**: Firefox 52.6.0
- **OS**: Gentoo Linux
- **URL**: riot.im/app/#/login

**Comment by lampholder** [comment]
Unfortunate to be late-triaging this one - sorry; unlikely that we'll be able to spot what caused this. It doesn't appear to be a common occurance, though.

@akater are you still unable to log in?

**Comment by akater** [comment]
@lampholder Yes, the error still persists. I have non-standard browser configuration but I'm usually able to log in via web. Hopefully, at least the error message could be made more specific.

**Comment by zebala** [comment]
I have this exact same problem, tried two different browser to no avail. Also can't login via phone app so maybe the user is broken? This was with a brand new account.

**Comment by jryans** [comment]
@zebala Did you create the account on matrix.org HS or somewhere else?

**Comment by zebala** [comment]
Somewhere else. Apparently it was a database hiccup on their end. Still, the error message wasn't that helpful :)

**Comment by jryans** [comment]
Sounds like at least your specific issue has been resolved, great! I agree this error message is not great, so we can hopefully use this to improve the message.

**Comment by kittykat** [comment]
Leaving this issue as "Help Wanted" as it would be helpful to have an improved error message proposal for the Design Team to review.

**Comment by richvdh** [comment]
I think this needs clearer steps to repeat in order to be actionable.
